docs/source/routing/operations/defer.mdx

```diff
@@ -1,137 +1,14 @@
 ---
-title: Router Support for @defer
+title: Configuring and applying @defer
```

Comment:
I appreciate separating out the explanatory material vs. the guide material! 🙌

Reply:
I started doing this when creating a separate "Feature Overview" left-nav section. That's changed though, everything's under "Features" w/out separate. Will stash these explanatory+guide pages for future improvement.